js/background.js

```diff
@@ -139,6 +158,10 @@
         messageReceiver.addEventListener('empty', onEmpty);
         messageReceiver.addEventListener('progress', onProgress);
 
+        if (!storage.get('migrationEnabled')) {
+            checkDeprecated();
+        }
```

Comment:
This runs every reconnect. Seems like you're gonna be leaking timers each time we hit this codepath. Shouldn't this be a once on startup thing?

Reply:
Absolutely. Updated.